main.go

```diff
@@ -20,15 +27,23 @@ func main() {
 		log.Fatal("Usage: tldr <command name>\n")
 	}
 
+	var page string
+	var err error
+
+	cache := newCache()
 	command := os.Args[1]
-	page, err := fetchPage(getPlatform(), command)
 
-	if err != nil {
-		fmt.Println(err)
-		os.Exit(-1)
+	if page, err = cache.read(command); err != nil {
```

Comment:
this comment comes first top-down but it's the least relevant. It could be fun to fire both requests in parallel so on a cache miss, the request is already in-flight (at a cost of always consuming some network resources).